cudapoa/src/cudapoa_add_alignment.cu

```diff
@@ -43,13 +44,14 @@ namespace cudapoa
  * @param[in] s                               Current sequence id
  * @param[in] max_sequences_per_poa           Maximum sequences allowed in a graph
  *
- * @return Number of nodes in graph after update
+ * @return Status code for any errors encountered.
  */
 
 template <bool msa = false>
 __device__
-    uint16_t
-    addAlignmentToGraph(uint8_t* nodes,
+    uint8_t
+    addAlignmentToGraph(uint16_t* new_node_count,
```

Comment:
Why not pass `new_node_count` in as reference?

Reply:
Passed in as reference now. More habit that I tend to move towards pointers for values that can change